### ISSUE TITLE
[Question] How to apply batch norm layers in A3C/A2C

### ISSUE THREAD

**Issue by sxyd** [ray-project/ray#5564]
### Describe the problem

I write a TFModelV2 subclass model for A3C training and the code is like as follows:
```python
class BatchNormModel(TFModelV2):
    def __init__(self, obs_space, action_space, num_outputs, model_config,
                 name):
        super(BatchNormModel, self).__init__(
            obs_space, action_space, num_outputs, model_config, name)
       
        self.dense_1 = tf.keras.layers.Dense(64)
        self.batch_norm = tf.keras.layers.BatchNormalization()
        self.dense_2 = tf.keras.layers.Dense(10)
        
        inputs = tf.keras.layers.Input(10)
        x = self.dense_1(inputs)
        x = self.batch_norm(x)
        outputs = self.dense_2(x)

        self.batch_norm_model = tf.keras.Model(inputs, outputs)
        self.register_variables(self.batch_norm_model.variables)

    def forward(self, input_dict, state, seq_lens):
        model_out, self._value_out = self.batch_norm_model(input_dict["obs"])
        return model_out, state

    def value_function(self):
        return tf.reshape(self._value_out, [-1])        

    def update_ops(self):
        update_ops = self.batch_norm.updates
        return update_ops       
         
```
When performing A3CTrainer, I will get a TF exception complaining about the observation placeholder not being set, and the log is like "You must feed a value for placeholder tensor 'Placeholder' with dtype float and shape [?, 10]".

If TFModelV2 subclass is able to be applied to A3C/A2C, please give an example.
